### PR TITLE
Create CO2STORE_GASWAT-02.DATA

### PR DESCRIPTION
**Pull request by OPMUSER**
Add another test for CO2STORE in combination with GASWAT annd VAPWAT. This test is used as a second example for CO2STORE in the 2023-04 manual.

Ready for review.

### CODE REVIEW

**Comment by bska** [comment]
@tskille: Would you mind having a look at this model, please?

**Comment by tskille** [comment]
Yes I'm happy to review this model.

**Comment by tskille** [comment]
Thanks for the test case. This is highly appreciated. I have a few comments that I would like you to address. I believe that you should strive to make the test cases consistent with what we actually support in OPM Flow.

**Comment by OPMUSER** [comment]
Yes, that's fine. Just let me know what I need to change and I'll update accordingly.

Note, I have used this test case as an example in the manual, so we need to get it right.

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+--       MXWELS  MXCONS  MXGRPS  MXGRPW
+WELLDIMS
+         2       30      1       1                                             /
+-- ---------------------------------------------------------------------------------------------------------------------------------
+-- NUMERICAL AND RUN CONTROL OPTIONS
+-- ---------------------------------------------------------------------------------------------------------------------------------
+--
+--       ACTIVATES THE FULLY IMPLICIT SOLUTION OPTION
+--
+FULLIMP
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:94]: C02STORE_GAWWAT-02.DATA was based on C02STORE_GASWAT.DATA, so the same comments apply to that that case as well.

Deleted.

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+--                STB/MSCF
+--       ------   --------
+RVWVD
+         1000.0   0.001
+         3000.0   0.001                                      / RVW VS DEPTH EQUIL REGN 01
+--
+--       RESTART CONTROL BASIC = 4 (ALL=2, YEARLY=4, MONTHLY=5, TSTEP=6)
+--
+RPTRST
+         RESTART PRESSURE SGAS SWAT DENG DENW VGAS VWAT XMF YMF WMF            /
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:271]: Changed to:

```
RPTRST
   'BASIC=2'  'ALLPROPS' /
```

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+--
+--       DEFINE SOLUTION SECTION REPORT OPTIONS
+--
+RPTSOL
+         PRESSURE SGAS SWAT DENG DENW VGAS VWAT XMF  YMF WMF                   /
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:276]: Will remove.

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+-- '*' 2 /
+-- /
+-- WCGMR
+-- '*' 1 /
+-- '*' 2 /
+-- /
+-- WCOMR
+-- '*' 1 /
+-- '*' 2 /
+-- /
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:307]: I would like to keep these, as the intent is to implement these summary vectors.

So no change.

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+--
+-- SCHEDULE SECTION
+--
+-- =================================================================================================================================
+SCHEDULE
+--
+--       DEFINE SCHEDULE SECTION REPORT OPTION
+--
+RPTSCHED
+         AMF WMF YMF DENG DENW VWAT VGAS                                       /
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:324]: Will remove.

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
+--
+--       RESTART CONTROL BASIC = 4 (ALL=2, YEARLY=4, MONTHLY=5, TSTEP=6)
+--
+RPTRST
+        'BASIC=1' 'ALLPROPS'                                                   /
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:329]: Will update

File: co2store/CO2STORE_GASWAT-02.DATA [hunk 7]
```diff
@@ -0,0 +1,380 @@
...
+-- WELL  FLUID  OPEN/  CNTL  SURF   RESV   BHP   THP   VFP
+-- NAME  TYPE   SHUT   MODE  RATE   RATE   PRES  PRES  TABLE
+WCONINJE
+INJ1     GAS    OPEN   RATE 100000  1*     300                                 /
+/
+--
+--       ADVANCE SIMULATION BY REPORTING TIME
+--
+TSTEP
+         1   36*30
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply co2store/CO2STORE_GASWAT-02.DATA:374]: Changed to:
```
TSTEP
         1   24*30
/

```

**Review by tskille — APPROVED** [review]

**Comment by OPMUSER** [comment]
Merging as approved by @tskille

**Pull request merged**